api/router.js

```diff
@@ -101,10 +103,7 @@ function campaignBot(req, res) {
       req.user = user;
       const signupId = user.campaigns[req.campaign_id];
 
-      if (signupId) {
-        controller.debug(req, `loadSignup:${signupId}`);
-        return controller.loadCurrentSignup(req, signupId);
-      }
+      if (signupId) return controller.loadCurrentSignup(req, signupId);
```

Comment:
I wasn't adding per https://github.com/airbnb/javascript#blocks--braces... 
````
// good
if (test) return false;

// bad
function foo() { return false; }
````

Looking to add #638 next to avoid syntax changes in future PR's.